docs/specs/openapi.yaml

```
@@ -167,13 +401,22 @@ components:
         status:
           type: string
           description: The text for the HTTP status code, see https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml.
```

We should be able to use Markdown for this. I'd suggest a couple of very minor edits:
```suggestion
          description: |
            The description of the HTTP status code.
            See the [IANA list](https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml).
```

---

I added a new empty line between the two lines of the description because I think you want to make them show up in different lines, right? Not just to improve the readability of the YAML spec. Feel free to open another small PR to fix this if it wasn't your intention.